### ISSUE TITLE
Harmonize OBI:solvent role and CHEBI:solvent

### ISSUE THREAD

**Issue by StroemPhi** [obi-ontology/obi#1771]
[CHEBI:solvent](https://terminology.nfdi4chem.de/ts/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_46787) seems to be equivalent to [OBI:solvent role](https://terminology.nfdi4chem.de/ts/ontologies/obi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBI_0302732), as it is defined as a role and used in other OBO ontologies as such under BFO:role. Unfortunately, it is not labeled as a role in CHEBI and its textual definition refers to the molecular entity that bears this role not the role itself. So the label and definition including the asserted subclassOf axioms in OBI are more accurate.

The benefit of CHEBI:solvent is that it is being used in CHEBI in subclassOf axioms on those molecular entities that have a solvent role. 

Now I'm wondering is how this should be resolved.
Should the two classes be made equivalent? Should there be an xref annotation?

**Comment by bpeters42** [comment]
Thank you for pointing this out. The class was created in OBI as the Chebi modeling was unclear, and we wanted to be able to refer to chemical entities that are part of an aggregate as sometimes being a solvent, and sometimes not. As you point out, that is the basic BFO understanding of roles, and Chebi has nominally signed up to BFO and OBO, but the Chebi-role branch in particular has been a mess for a long time. Our attempts over the last decade of working with Chebi have not been successful, as the people in charge there consider it more of a database system. 

Personally, I don't think there is much hope of addressing issues like this until either Chebi signs up to being part of the OBO foundry in more than name only, OR there will be an alternative that is adopted instead of Chebi that is more open to collaboration. 

If I am sounding bitter: I am! We paid for a Chebi curator for years from IEDB funding, and still our requests for them to see interoperability with other OBO ontologies as a goal were ignored...

**Comment by StroemPhi** [comment]
Thank you @bpeters42 for answering. I understand and partially share your frustration. However, to close this issue, I'm still wondering, if it might be good for documentation to link from OBI:solvent role to the CHEBI:solvent in an annotation (e.g. comment, editor note or term tracker item to this issue) so that this discrepancy becomes more clearer?

**Comment by StroemPhi** [comment]
Sounds perfect to me!

**Comment by cmungall** [comment]
To carry discussion from the PR.

[CHEBI:solvent](https://terminology.nfdi4chem.de/ts/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_46787) is clearly in the role branch and not the chemical entity branch in CHEBI, and it has always been this way.

The primary problem here is with the labels and the text definitions, I think this was @StroemPhi's point. `solvent` doesn't advertise itself as being a role (though it is a reasonable guess, and the alternative `solvent role` is an ontological construction that only has 1k hits in google scholar). The definition genus says `liquid` which makes no sense but this is not the worst issue with chebi definitions.

**Comment by StroemPhi** [comment]
@cmungall, yes this was my initial point. Although I assume CHEBI:role and CHEBI:chemical entity to be disjoint from hwo they are used in many other ontologies. This isn't asserted on the artifact behind the CHEBI PURL when loaded into Protégé.

**Comment by StroemPhi** [comment]
When we subsume these two CHEBI branches under `BFO:role` respectively `BFO:material entity` in our ontologies, we have the disjoint axiom from there and are logically fine. Still, the label and textual definition will be distracting for our users and we would have to inject annotation axioms to fix this. So maybe changing the proposed editor note to something along the lines of the following would be better?
"**The class CHEBI:solvent is logically equivalent to this class, iff it is properly subsumed under the BFO:role branch. The label and textual definition of this CHEBI class are however wrongly denoting the material entity which bears this role and not the role itself.** The OBI modeling was needed as what is considered a solvent in a mixture can depend on the intent of the person creating the mixture. It would be great if this could be dealt with in CHEBI itself. See also https://github.com/obi-ontology/obi/issues/1771 "

**Comment by cmungall** [comment]
“Wrong “ is way too strong for describing the label axiom IMO!

**Comment by bpeters42** [comment]
As I wrote in the pull request, I cannot reconstruct what I looked at when writing my first comment in this thread a month ago, where I saw specific material entities as children of 'solvent'. I may have looked sloppily, or may have looked at a hierarchy that was displaying incorrectly. Chris confirmed that there has always been a clear distinction between roles and material entities in Chebi that is compatible with BFO and therefore OBI. 

So looking back, I would make a 180 change, and advocate for the OBI class to be mapped to the Chebi class and be considered equivalent. 'solvent role' is clearly in Chebi scope more so than OBI. Fixing the textual definition in Chebi would be perfect. Until that is done, we could instead place a comment on the OBI term to say that this is essentially the Chebi term, and we are planning to retire it.
